.goreleaser.yml

```diff
@@ -104,25 +104,39 @@ release:
   disable: true
 
 dockers:
-  - 
-    goos: linux
+  - goos: linux
     goarch: amd64
     dockerfile: Dockerfile.release
-    binaries:
-      - nri-prometheus
+    ids:
+      - nri-prometheus-nix
     image_templates:
-      - 'newrelic/nri-prometheus:{{ .Major }}.{{ .Minor }}'
+      - 'pgallina/nri-prometheus-test:{{ .Version }}-amd64'
+    use_buildx: true
+    build_flag_templates:
+    - "--platform=linux/amd64"
     skip_push: false
-  - 
-    goos: linux
-    goarch: amd64
+  - goos: linux
+    goarch: arm64
     dockerfile: Dockerfile.release
-    binaries:
-      - nri-prometheus
+    ids:
+      - nri-prometheus-nix
     image_templates:
-      - 'newrelic/nri-prometheus:{{ .Version }}'
+      - 'pgallina/nri-prometheus-test:{{ .Version }}-arm64'
+    use_buildx: true
+    build_flag_templates:
+    - "--platform=linux/arm64"
     skip_push: false
```

Comment:
Could we add `arm` as well here? I think we're publishing other integrations and the agent for arm(32).